phpBB/docs/README.html

```diff
@@ -91,24 +91,25 @@
 		<div class="content">
 	<p>Installation, update and conversion instructions can be found in the <a href="INSTALL.html">INSTALL</a> document in this directory. If you are intending on converting from a phpBB 2.0.x or 3.0.x installation we highly recommend that you backup any existing data before proceeding!</p>
 
-	<p>Users of phpBB 3.0 and 3.1 Beta versions cannot directly update.</p>
+	<p>Users of phpBB 3.0, 3.1 and 3.2 Beta versions cannot directly update.</p>
 
 	<p>Please note that we don't support the following installation types:</p>
 	<ul>
-		<li>Updates from phpBB 3.0 Beta versions to phpBB 3.0 RC1 and higher</li>
+		<li>Updates from phpBB 3.0 Beta versions to phpBB 3.1 RC1 and higher</li>
```

Comment:
or should it still be 3.0 RC1 and higher? Because 3.0 is EOL i thought, that is a necessary change.